src/components/KeyboardControlsProvider/KeyboardControlsProvider.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import KeyboardEventHandler from 'react-keyboard-event-handler';
+
+const emptyFn = () => {};
+
+const KeyboardControlsProvider = props => (
+  <KeyboardEventHandler
+    handleKeys={Object.keys(props.commands)}
+    onKeyEvent={(key, ev) =>
+      (props.commands[key] || props.commands.all || emptyFn)(key, ev)
```

Comment:
All won't fire on all events, if theres a specific one, seems incorrect. Is `all` a fallback?

Reply:
do you have a specific example?

I've just pushed a fix to normalise the key naming conventions.